src/components/Meals/AvailableMeals.js

```diff
@@ -54,16 +56,23 @@ const StyledMealsList = styled.section`
 
 const mealToElement = (meal) => {
    return (
-      <li key={meal.id}>{meal.name}</li>
+      <MealItem 
+         key={meal.id} 
+         name={meal.name}
+         description={meal.description}
+         price={meal.price}
+      />
    );
 };
 
 const AvailableMeals = props => {
    return (
-      <StyledMealsList>
-         <ul>
-            {mealsList.map(mealToElement)}
-         </ul>
+      <StyledMealsList className={props.className}>
+         <Card>
+            <ul>
+               {mealsList.map(mealToElement)}
+            </ul>
+         </Card>
       </StyledMealsList>
```

Comment:
`mealToElement` function does not really do anything. 
Can change it to following to simplify it
```es6

<StyledMealsList className={props.className}>
  <Card>
    <ul>
      {mealsList.map((meal) => (
        <MealItem 
          key={meal.id} 
          name={meal.name}
          description={meal.description}
          price={meal.price} />
      ))}
    </ul>
  </Card>
</StyledMealsList>
```